File: WebView/src/app/Classes/Projects/project.ts
import { User } from "../user";
import { Unit } from "../Unit";
import { Report } from "./Report";

export class Project{
    constructor(public Id : number, 
        public Label : string, 
        public Admins : Unit[], 
        public UnitsCreators : Unit[], 
        public DefaultExecutor : User, 
        public HeadOfProject : User, 
        public Reports : Report[]){}
}

File: Mobile/Mobile/Models/Project.cs
using System;
using System.Collections.Generic;

namespace Mobile.Models
{
	public class Project : Entity
    {
        public string Label { get; set; }
        public List<Report> Reports { get; set; }
        public List<Unit> UnitsCreators { get; set; }
        public List<User> Admins { get; set; }
        public User DefaultExecutor { get; set; }
        public User HeadOfProject { get; set; }
    }
}

